pkg/api/v1/types.go

```diff
@@ -3024,6 +3024,8 @@ const (
 	ResourceServices ResourceName = "services"
 	// ReplicationControllers, number
 	ResourceReplicationControllers ResourceName = "replicationcontrollers"
+	// ReplicaSets, number
+	ResourceReplicaSets ResourceName = "replicasets"
```

Comment:
I prefer @mfojtik's approach of moving this to the API group from whence it came.  I'd like to establish a pattern that federators can use when adding their own group.  `groupName/resourceName` seems reasonable.